rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -147,7 +171,7 @@ private CachingRlsLbClient(Builder builder) {
     }
     RlsRequestFactory requestFactory = new RlsRequestFactory(
         lbPolicyConfig.getRouteLookupConfig(), serverHost);
-    rlsPicker = new RlsPicker(requestFactory);
+    rlsPicker = new RlsPicker(requestFactory, rlsConfig);
```

Comment:
There's a lot of data in the `rlsConfig`. Let's pass _just_ the lookup service, so people don't think the picker needs the other information/begin using the other info directly in the picker.